lib/fragment.js

```diff
@@ -62,6 +64,10 @@ const getAttributes = (tag, context) => {
         fallbackUrl: attributes['fallback-src'],
         timeout: parseInt(timeout || 3000, 10)
     };
+
+    delete finalAttributes['fallback-src'];
```

Comment:
this would trigger deopt, why do you need to delete this property?

Reply:
good point. fixed in https://github.com/zalando/tailor/pull/323/commits/920f6e2a51678c3e73a929fc60e3fc5f1af15e6f .